containers/containers.go

```diff
@@ -91,6 +91,10 @@ type Store interface {
 	// List returns containers that match one or more of the provided filters.
 	List(ctx context.Context, filters ...string) ([]Container, error)
 
+	// Stream returns containers that match one or more of the provided filters as a
+	// stream of results.
+	Stream(ctx context.Context, filters ...string) (<-chan Container, <-chan error)
```

Comment:
Should probably define the error semantics for per container, as well as stream shutdown.

Reply:
I'm thinking, from our discussions, we will just hide this behind the `List` method.  We never really wanted it to be a constant stream of containers, just an efficient way to return many containers with grpc.  If I hide it behind the `List` method then we will just do the fallback if the stream rpc does not exist.